lib/internal/Magento/Framework/View/Asset/Repository.php

```diff
@@ -167,6 +162,52 @@ public function updateDesignParams(array &$params)
     }
 
     /**
+     * Get theme either by id or path based on whether the passed $theme is a numeric value or not
+     *
+     * @param string|int $theme
+     * @param $area
+     * @return \Magento\Framework\View\Design\ThemeInterface
+     */
+    private function getTheme($theme, $area)
+    {
+        if (is_numeric($theme)) {
```

Comment:
Yeah, looking into `\Magento\Theme\Model\View\Design::getConfigurationDesignTheme` this seems to be the best option to me (although we won't support string theme identifiers consisting of digits only).

Can we eliminate this method and still pass PHPMD? `getThemeById` and `getThemeByPath` seem much more understandable to me at the place of call but I would like to avoid method named like `getThemeByIdOrPath`.